ctapipe/io/tests/test_event_source.py

```diff
@@ -97,14 +96,14 @@ def test_from_config_invalid_type():
     EventSource.input_url.default_value = dataset
     config = Config({"EventSource": {"input_url": 124}})
     with pytest.raises(TraitError):
-        EventSource.from_config(config=config, parent=None)
+        EventSource(config=config, parent=None)
 
 
 def test_event_source_config():
     dataset1 = get_dataset_path("gamma_test_large.simtel.gz")
     dataset2 = get_dataset_path("gamma_test_large.simtel.gz")
     config = Config({"EventSource": {"input_url": dataset1}})
-    reader = event_source(dataset2, config=config)
+    reader = EventSource(dataset2, config=config)
     assert isinstance(reader, SimTelEventSource)
     assert str(reader.input_url) == dataset2
```

Comment:
what is this test doing? dataset1 and 2 are identical, so what does the `assert` statement check?
(Independent of this PR)

Reply:
moved into #1517